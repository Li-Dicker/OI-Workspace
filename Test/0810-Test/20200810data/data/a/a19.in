10
196
60155 72616 42137 65065 7137 90093 56618 27675 9819 62574 82537 76441 89857 27397 84196 11260 52633 83570 31770 88942 52447 61719 47342 95476 10986 34803 31925 73205 89012 78657 55841 42497 44123 6769 77055 4341 56897 16412 92781 35738 79971 24861 441 74476 18636 27039 32164 38630 17530 77817 52641 64087 47726 85545 58885 38976 73875 33615 8968 8236 68837 54503 14048 80029 7519 64545 66850 40218 50164 96570 38555 42649 70461 90806 23877 54439 53737 37379 68802 61224 71093 25814 35661 17253 3391 74712 40755 26065 7671 27298 7379 2151 13195 48101 33564 87525 49783 88847 46554 93854 70594 6830 35844 56681 6279 32222 70541 90731 66522 89637 87586 81762 48664 80344 87929 46709 68413 80914 75816 76572 76630 94168 56413 3097 29355 93808 47149 6968 57340 18003 59174 66654 42568 7082 77404 47047 53120 32260 79564 39694 34629 62495 3428 50439 82693 50087 31224 96505 51018 34410 99469 66225 61399 28943 96655 46209 40977 83545 46823 67363 29630 60453 8655 15905 15278 4695 3454 56484 41032 29965 3092 79109 27723 83795 11503 1060 16960 25667 70951 59848 5233 23039 61827 13173 26233 41028 93507 45168 91473 84452 35940 9011 53080 49466 39685 6869 
33508 17350 32837 26014 12184 29604 37068 7966 35722 28636 40818 48626 48751 9216 10639 34623 24258 6880 12752 23723 3056 46522 49290 22049 35162 13834 47977 15507 7772 41963 25659 19531 31161 14932 17962 1853 28396 28745 46628 41155 29536 23839 39150 5070 29891 44801 2999 39188 23581 21069 815 36725 29729 36636 22212 37577 3908 3574 25481 17626 22356 21713 30443 27432 14700 16925 14016 29170 9959 48814 42945 36354 12322 6922 15349 32764 44446 46307 46127 25338 49425 9328 30920 7609 37855 27691 19566 47712 1217 46722 39204 29429 41649 10496 16441 12611 32291 12315 12537 90 4278 5683 28983 12905 6693 2370 43384 18962 48783 24455 36414 13776 19581 20645 21881 45016 14536 31770 20174 39725 38314 32574 32461 37171 8596 30779 47977 49590 46189 42853 23737 22533 28800 23183 32844 48059 20085 36063 39315 28847 5196 46506 15464 4555 22169 33679 19077 34142 32733 17165 48023 29145 21579 34641 44060 49131 17872 28749 1815 17346 17256 6825 31273 23090 7510 25118 45073 8039 12693 16671 33346 35561 40406 10763 43788 41797 34281 6716 13372 3974 1590 44264 19518 32895 13655 4638 9712 6377 37259 44108 14327 42096 26400 24266 3239 28330 
196
38291 69923 85064 71498 33292 64793 16791 99692 22983 73448 70569 87011 39673 71966 6676 79862 46777 4891 97768 77018 91657 29577 3993 54524 16501 31093 74122 92331 95014 76469 92708 99948 31273 29354 44855 98247 33253 29305 17820 60047 66861 37318 98295 1796 34324 54367 59576 61773 89398 42371 25837 41132 13809 95502 67466 42366 63344 62595 80283 66505 418 88831 77293 25272 82809 48000 18956 38078 56486 64675 56035 88521 3838 20239 73840 34614 34599 43697 34111 6237 2736 32654 24087 15155 89655 52327 56946 43056 86415 61324 56036 77341 80047 84431 3773 77986 17946 20515 18145 64398 71294 47855 40887 13103 40414 83140 44849 97416 12897 91615 11047 10881 92329 44145 71970 59706 31840 19649 22849 30577 98726 63636 21078 82814 78367 19746 31723 98361 57175 85673 98986 73727 31302 42782 70510 17670 27187 11774 40895 23603 58015 98922 58152 61995 49018 63318 1472 38311 34935 71300 68126 91956 93520 23944 14858 91738 62961 69313 958 13649 99517 74275 36110 42849 3116 28105 18208 97430 36205 99839 40711 94044 78496 81135 21758 26102 78341 25795 69400 6545 37614 56279 10424 66727 52723 18127 91537 25957 47194 92984 34905 33037 25937 95687 22227 1068 
34210 16278 5642 39670 42320 25658 47635 36911 5114 48354 13440 3762 39332 44660 43403 8971 42331 25532 17575 35583 2361 12454 30463 35459 32897 13174 18077 9723 25903 15367 13351 39708 34884 272 45665 22342 13698 41966 45203 42445 37779 47089 43842 43612 21655 24837 37807 2900 3889 36794 15782 2133 21531 2627 3669 22210 22008 21 43675 12272 10273 38630 45996 23760 5866 11031 10723 6813 46611 31172 18036 7718 32261 11173 8457 44286 2691 2227 36486 8532 49418 20634 135 13386 46237 11202 34265 38130 33162 33571 42508 8900 27737 34242 49185 32414 30024 17775 44727 48877 9769 38735 6607 15043 25882 1584 42901 29867 27374 6392 26434 26469 3179 32854 46591 13412 49003 3263 21351 42809 32657 47546 46403 16787 5273 2811 20238 21782 2995 34932 22627 5718 7793 16322 6175 47843 49803 24459 15581 35463 34427 15222 24319 27356 17694 32346 43543 11793 27544 26689 166 39298 25769 6710 23046 48269 19885 49459 46873 30441 9347 9675 7264 33133 34558 31438 27563 8779 6880 3192 4510 6876 21900 3462 25066 21094 47205 15610 13829 35369 9198 24517 14994 5791 19498 2389 22914 32943 11018 35841 41942 12806 4018 23667 36008 10353 
200
18526 25246 32293 18589 21625 37883 26710 27378 79037 20766 1201 86437 22586 45606 47782 47776 20300 11538 1489 61330 45180 71385 61812 57758 16499 33853 77452 2073 24683 80420 8729 28250 41311 1754 18836 12864 81550 90019 26828 33070 71997 88524 14569 16357 30508 45977 68628 66512 29638 56753 5533 16510 79643 47641 70536 11307 53835 64160 39984 66563 34465 29356 10673 70422 67274 5511 31388 89982 77694 95637 60926 14036 45402 77196 62519 33252 90915 20218 20421 4627 61686 86366 4713 85147 78941 24118 45685 45637 16070 17242 3455 27797 22578 10827 68183 82174 85342 74686 59781 574 22228 67007 86659 68995 8651 79892 63789 11618 74292 4452 43484 79635 77405 17355 33203 96680 2964 26341 58170 67648 89733 96087 56975 70170 31560 38065 69283 21981 35283 65523 85889 82277 86616 42909 76394 14724 92599 6112 46774 76798 34751 85657 63424 58206 28137 43550 48305 96939 93530 94530 81798 35095 43258 18390 40496 25521 92630 2864 47778 97482 76635 50657 68458 90796 50258 68381 51642 10436 4978 17897 62985 34378 75562 5011 60069 61217 43773 19753 80800 11985 72699 21441 51334 15371 28708 21603 80956 42082 51701 54602 53863 1597 47467 22478 23828 60389 82938 82203 17539 9053 
3876 32330 2008 38444 34830 6766 38743 23774 1876 38279 37472 35194 34457 9671 5495 1169 41942 43603 10037 10997 3648 20957 31741 32309 46488 17380 36107 46720 12216 31641 21457 33284 21665 35669 35158 33344 30596 30097 31634 17512 10116 6037 12 49510 15699 45470 6151 45103 35805 44057 3803 17435 42996 45657 45992 40053 49162 17526 34403 32211 31359 13488 19372 27138 28147 7763 12387 44202 28547 7411 8546 9560 30678 30683 42426 14920 43604 21849 40561 14044 12154 42672 15978 33902 21438 22724 7197 49831 24164 26391 17230 49410 40988 41193 9842 22151 47826 41553 24088 890 13166 19608 24663 36684 22191 45190 28180 37499 11667 28675 38410 17839 49306 8248 1765 34734 5554 40269 8824 13221 30823 20040 1872 5082 40063 27361 10833 42980 39692 22070 45089 30143 28313 5494 21031 48038 47798 28022 5235 23290 7130 34067 4522 34959 41413 23400 18090 13224 11898 40912 25381 38010 29860 24519 5917 43558 36920 28395 1331 23608 28248 49859 20594 25949 25703 10083 35602 24154 28547 9036 16712 8018 22 45122 37235 23551 18490 17648 6315 18547 10166 7945 5147 25773 5768 45546 13881 2951 47753 19321 43071 44927 2135 49232 31666 47810 22613 4950 48425 17307 
197
51314 57420 56356 49161 55262 61179 35697 32039 91033 58004 91177 13171 13377 2278 41180 525 65949 89264 80724 61277 48542 13600 39417 2574 22119 55497 78930 16889 30515 85233 86812 69284 162 72751 32543 49313 81801 24674 65460 71505 74295 16310 55351 94261 89705 70588 39801 35286 19793 14172 15872 72641 97428 30131 45993 11989 50348 23927 66282 17773 91751 84284 35061 14675 93984 69148 83856 56372 51250 54554 72408 45330 89172 6119 77245 99609 9659 77255 56603 51488 1955 9873 88381 27480 4081 71870 33138 48415 13675 15637 88651 50449 28213 17012 23314 65599 3582 61249 35498 49254 56217 8709 72464 5073 77798 89054 18928 81144 69947 85948 72098 70324 60365 37026 61057 83502 28117 42729 56056 997 93501 36587 18001 40572 5298 20836 38584 82530 16153 24647 23996 61095 5717 97552 18911 18687 3758 65774 58868 39952 81983 73350 83175 22639 88717 41595 80303 11425 59005 52881 79590 76970 61607 97318 40979 31876 18270 46169 22100 78481 42026 79041 43486 36345 56212 6679 20934 30989 69666 2960 59135 29626 8024 30840 35883 36382 74448 53154 82898 45 56359 4556 86670 68855 70006 2092 27850 17251 96377 64876 91962 13927 1163 96663 99403 39210 85629 
24695 41992 4299 41542 45868 32816 39994 26634 4385 40717 34913 40630 8952 20749 15091 10440 6191 47807 32210 42784 17425 39228 22341 21183 22973 3622 48664 26642 41044 12146 18508 13623 13104 14219 7075 39306 23032 34282 10736 46161 34271 8610 7889 19097 28656 5097 42376 40507 36621 40134 25689 1450 25347 13200 26504 16472 14132 26479 16795 6112 45676 10510 47897 44093 21453 4124 13787 7155 20536 43273 39986 34268 20174 2878 33371 15076 44961 45490 34850 33612 35002 39268 19540 6396 42985 30703 19068 47204 42048 39660 35249 5807 22127 38159 8212 7234 4704 42723 20063 34699 19886 28319 40285 47784 9095 21408 34910 16398 43828 16328 47094 36592 10389 27199 32138 12045 37006 43729 5669 45737 45116 23095 15852 20487 3938 7774 45431 30560 40175 44049 49792 11523 22914 18696 36899 8714 24594 32409 34334 37172 9493 38596 43011 10424 39101 2847 11533 5889 17870 1182 46735 39918 28940 41208 23529 43191 40211 48970 21034 30230 35944 37698 22068 8587 45591 45353 37883 6654 34019 47883 376 18344 43453 19463 25482 44013 24006 6133 8866 37539 13246 42641 18250 36944 31918 6694 24594 35910 44133 18192 48233 10288 31744 25009 31130 31009 36630 
196
10958 92594 97186 77816 71016 20216 39232 98205 74245 59564 78513 71190 93410 3084 73002 34132 27521 29669 17378 78015 62979 64729 35371 13892 10537 28699 47223 75054 22579 95227 80329 27794 41701 7596 33113 95974 79869 28243 590 1948 7982 63405 75831 25307 94556 25843 88801 9125 88281 95431 83211 7611 32130 70779 55137 93803 33160 21134 6986 51564 30113 49612 7782 86396 58862 1198 14137 99614 24840 13947 54625 67824 1012 19282 54841 64829 41294 58485 49258 78673 65719 18896 7743 5607 47599 13304 69071 49311 86668 5793 6738 88569 95246 33763 27779 81338 10685 17634 19041 37234 38379 53244 66015 7110 85910 93599 85394 93193 50401 61252 63876 74544 52410 54175 76257 58104 3766 76417 48368 96520 84109 91728 52251 99399 46326 48887 54636 81977 99608 55339 13576 44321 60889 31681 35104 83614 52830 6962 12543 35303 82770 10792 29139 12104 53940 71755 31628 81074 55970 90006 73844 43572 87690 4360 72409 12079 50747 15973 27486 48018 37729 56865 40379 72769 86993 21810 19834 36323 6641 86254 75053 60135 56874 34580 87319 57574 95650 25746 90226 98446 7029 65889 39344 72104 43239 82148 7449 79451 31800 82 62345 19413 27500 20648 65218 21915 
33119 16082 28863 15164 20153 11815 34217 7799 9359 40128 46919 21078 30971 12943 19571 42455 21737 47481 14578 32668 9407 44628 12468 261 19680 1398 11572 4760 45235 7516 44967 10500 49224 13379 9135 28362 44582 14595 214 38618 7233 27212 46509 33694 9551 30872 16349 34114 45532 37042 28929 47488 48861 10919 34999 16450 33233 45169 45869 3618 48970 42059 8288 19821 4781 34875 39332 41101 9878 19058 183 442 40567 18363 36584 47489 5820 49782 29892 41012 48004 843 46396 5123 15642 48916 7437 9694 31475 4727 38335 42796 27580 23121 32970 8012 43439 3733 12754 9944 21393 45933 10849 16054 29202 49107 14594 26292 33816 30764 22472 7311 31426 6842 38929 6652 1681 8273 13441 42421 42878 39765 41933 32334 5428 26608 42981 9215 7445 10656 28912 42851 26570 13132 24495 26857 44706 1875 39015 5958 48527 20721 6335 17848 46679 2621 30482 42078 27168 49725 13223 2245 36588 17874 36185 21202 19549 447 37148 47964 34263 47740 20057 32269 38361 10039 9913 22866 30386 13615 43461 6550 149 12091 13777 49552 28803 24787 13354 46943 48148 33974 15554 9531 31775 28397 27448 39791 30190 45379 47374 3581 8211 14984 26399 40971 
197
6276 1750 71195 71864 7259 87622 98095 75345 68549 63295 87960 39484 64500 98363 64605 46785 42873 9585 66116 3462 76650 31038 56600 41486 3874 89977 95041 47382 84327 57439 80512 21114 49203 34044 81297 95490 34922 8735 27789 93137 47730 83877 15374 23015 18236 6068 439 93512 32301 9503 21566 88496 45782 12128 28113 81123 21933 34412 11228 42428 67862 10439 18016 56587 2752 16546 78922 52842 38656 8226 21642 18872 55240 6592 37100 99709 93798 76210 43450 73973 40237 40757 25778 18113 15443 77481 63021 77333 45485 46085 21162 56557 68641 86330 98447 63459 22936 46697 90633 33589 7620 5340 51151 19442 23475 69110 18263 59952 53355 97665 62494 31448 65350 41752 85788 81745 17440 23448 70421 6882 82356 39773 65380 84977 22569 46053 29941 53680 80475 10604 3964 33661 17948 35696 28536 50420 89430 91040 81569 71006 73761 74020 78947 23912 19322 8472 92269 57487 39476 92167 43021 66309 12181 76053 73087 9881 49158 37524 76520 29568 33072 38477 48844 42728 58801 84090 90846 83756 83820 11750 93823 24940 51650 30298 33955 89586 2508 79323 52253 38130 44445 43842 57135 97931 6644 787 53748 60929 54905 57815 18613 39220 18683 49192 37138 22251 4626 
27660 19202 29468 28568 27176 20778 24025 22004 30356 31278 12556 28972 3997 5546 21293 24576 29336 32209 33559 38528 42552 11529 48631 14357 21566 17650 47795 7505 25221 19546 13533 35601 24870 15682 49398 5145 10113 44940 219 8061 42806 36142 47515 18329 32421 47911 28975 29500 31505 36110 6665 6669 46153 25531 7185 15943 49392 25470 2265 42265 17567 4944 14835 37236 9911 23090 42663 45477 26253 33630 18287 5980 9099 10924 11913 18780 28858 41531 41715 23481 44221 22722 49472 13610 27409 37555 11900 42380 39646 29482 48060 9726 317 9238 14416 14343 13740 4979 26794 27245 20372 1778 20906 32078 41231 25047 32273 38718 20494 43094 42199 27291 7070 14224 10183 32918 21073 30462 15452 23426 12623 21857 45439 6564 6165 27394 11196 44625 1804 35925 41393 12296 29621 45689 25286 24741 35355 8959 38482 9977 37934 31310 24138 30375 9664 23473 43214 25714 24345 41951 15726 7469 46853 47857 29526 29425 3996 23283 19931 32337 14410 6062 18104 17266 27056 13058 33407 29936 15884 11737 19479 12997 44243 23864 1289 36271 8818 19531 4373 28314 39209 7997 24699 11849 12627 47950 4401 29796 26505 38756 11589 6880 1469 35152 8949 32283 27781 
198
72046 62454 37035 69939 61934 33926 42060 68390 59581 38725 82214 34557 80042 82532 20627 9478 22185 25459 12099 19203 99530 91860 33541 48572 86442 80318 3087 78341 74741 43818 27200 22541 85223 68422 21184 49490 61119 80184 33409 17218 74318 21880 65613 78308 8466 90802 58281 61386 72674 12807 71220 67254 41449 2510 4858 41001 9385 91847 74756 33425 16661 57194 96533 25919 60569 92305 94615 89990 80513 14214 82804 18597 7172 3145 66034 40175 47247 90624 20717 84184 16448 36943 64445 65468 99695 96382 63798 28621 82173 1196 440 42218 60604 67813 76265 53072 18633 97397 74205 55877 67303 14616 2653 47211 40389 74725 12405 44598 1868 15532 70242 43639 90692 79476 18530 89843 1937 94716 91458 28761 60779 41552 60153 96147 92526 17389 41944 12412 4795 6270 53699 65810 95169 78061 83935 30266 3674 71243 30761 20588 66503 80994 88915 85751 14511 61053 2649 58795 26678 66232 93750 78521 48643 91864 43540 9770 40820 96818 42622 92600 80808 49452 82742 48082 18036 47001 66783 44127 12041 84905 18944 59818 44956 51630 35664 69548 49423 96053 43554 50020 36039 30494 81163 60763 67291 21445 97846 76199 91462 82279 88307 95292 83614 47816 29067 52439 68398 52555 
3709 35364 23918 30575 8014 28257 37529 11254 2182 47745 13874 21959 45144 47487 39983 22419 2805 39399 15346 45374 45234 20268 28863 48964 3696 27016 6133 43330 24773 12876 12263 46531 31494 3682 44992 11844 21960 13410 39941 37780 7998 35961 3714 38472 46958 34539 19854 5803 20393 48856 18514 32201 7496 34919 30275 35873 30002 18127 9685 23905 13619 34362 14886 43038 4596 10026 48805 40057 39451 35522 35264 29692 41880 17418 1577 22384 21471 10839 3607 8683 12286 36774 34270 24166 20572 13178 46171 29483 21644 20157 45556 35268 49650 44168 6062 32081 955 22515 21250 45382 7754 30322 45958 41003 24916 1970 18134 47017 39036 21081 41598 8153 4004 45438 38483 27301 35497 4220 18637 48620 27273 26421 34463 45530 31865 26968 960 33849 6137 40314 43711 32850 34838 11797 28923 18098 42618 13609 48272 45295 36933 38218 15225 12679 21089 31222 49121 47332 34727 14795 8389 48298 28278 33659 20572 49034 42339 26760 47539 9202 35664 19125 39666 35747 36904 29795 37966 29106 30189 1140 41661 45354 23095 28992 7611 44540 32877 33294 24819 1008 19299 25427 7513 11457 49844 16535 12183 46658 8352 15799 42036 17002 1633 48815 4332 7404 4684 35883 
199
29266 24851 32706 31074 7899 78671 29816 2265 17573 64300 17401 23738 18581 68157 42265 37368 97417 79525 27026 50218 10282 31564 99258 18823 99570 40620 24742 72025 22872 84854 84327 79069 38207 39949 53199 10040 67215 51708 75809 74339 2035 4267 61777 27129 37225 59688 67324 21071 78080 85721 59845 46966 61439 53934 52113 29857 99933 15168 1076 45088 4222 83227 48576 24265 40503 27688 72877 40008 2148 56602 89670 25158 61894 7406 20213 39703 29059 34423 34967 43467 72040 49180 93820 27639 83770 9359 99563 92251 15772 8491 75736 132 7208 33603 5634 48370 81833 26965 1475 64893 2849 98169 99559 71283 45263 49496 35933 40545 62289 30683 48062 57997 34870 80869 15793 83856 13445 43621 48571 29079 65556 65040 44707 24438 75463 49718 95957 45771 22590 22475 34017 39927 22107 97600 18938 42286 82150 19330 49264 31885 54309 42672 22475 32779 31781 37531 30667 77363 68565 15514 89697 76415 61461 81803 79162 93969 34559 66737 79749 25565 75686 77480 52514 50232 84534 21888 82711 47986 56816 89310 74021 60042 42234 75927 97203 14498 52794 97348 59735 32547 92040 39732 95442 21183 23649 41059 73404 63237 26448 96715 73118 11548 99831 83413 83076 51039 41238 93922 68194 
10185 40623 20049 38783 21350 44135 26193 47651 19907 48473 30527 29838 45675 17374 24849 47814 5665 3958 11690 18903 6343 38267 24876 4173 44208 20896 23891 15846 13913 32146 11971 32378 34018 30556 38824 42452 2729 44152 14926 18635 40167 9233 15172 2768 33755 41690 43967 10020 8091 29607 44009 20020 37949 37230 32761 11026 14628 35242 46439 11389 37656 16477 30202 29832 39724 47853 3237 43587 30778 47115 48276 42028 6471 14133 36957 1439 46299 33177 44872 1335 13490 28605 25982 31329 16269 42720 39967 25100 38036 12833 41171 29521 24868 24662 1836 1559 31889 44843 6844 12463 14576 3641 41525 39510 35465 44842 6307 42651 26492 20373 43904 25847 4009 7096 39443 23804 38664 21914 32823 43886 10054 23621 13025 34670 47386 15584 425 47811 24426 28698 19758 16269 764 14372 45815 29619 1745 15771 27163 11879 37568 33208 17008 40732 34957 13648 39669 17026 10402 18220 14866 45971 23213 15810 2803 28824 19466 22989 4031 32695 6388 31448 2766 4931 13957 29926 23311 7452 33008 33757 14766 29041 28945 7823 27409 46962 42965 46803 49193 19553 41263 9815 22427 11613 20018 32767 24746 35712 2225 13693 12737 26808 12036 7148 46334 30102 5610 35410 48623 
198
9671 57683 94805 87905 74356 87722 15197 45685 3888 6663 29580 46854 54812 10568 89852 64074 59903 99073 25299 61042 55935 18452 42698 52100 40297 77279 8431 78943 20426 90939 84507 60840 63481 43308 51594 69375 22850 77284 51592 55800 71674 92360 48231 66762 94685 10603 91348 15948 36653 20237 44448 40809 67957 8624 50443 72266 32795 56899 79229 24920 38125 8869 34343 47989 1632 42781 93600 66573 81654 76101 42476 52588 31627 24510 21243 91857 79412 71986 87524 68260 19361 173 9443 75165 61966 23033 15413 6097 80408 90663 44159 76033 51501 93605 52319 85826 86150 37016 62555 49985 17974 98465 8738 3766 31797 6200 12504 45884 62229 89974 14685 73070 40418 67499 6342 77256 85658 74034 92710 88303 6594 58765 73625 33133 25376 48599 20762 142 92084 92234 49335 39273 53239 33839 82589 57299 34940 89676 33381 68980 57326 79580 65054 34961 19201 58787 27477 45519 3104 3865 58880 76723 16775 19950 18275 46232 78745 29827 71161 36487 78079 58262 30137 4923 61413 25921 17684 10309 56338 48617 78863 53799 52403 68829 78298 35402 62075 66528 65662 24186 58622 90436 46373 45589 36106 56095 84658 80039 78111 7959 51395 22347 29306 66858 95839 2728 62224 33087 
6069 23521 30829 39870 40696 17744 10669 21370 12296 9762 109 7712 34800 2024 35392 11996 22947 5925 49594 1792 16835 2923 37207 12359 47054 38693 33442 19330 37930 45896 9149 24448 273 27750 23204 35421 2452 28709 20905 38508 20118 37939 37390 47822 8304 24248 26381 48819 3172 15384 14295 20444 13837 4660 37901 45131 12337 25809 590 48541 35795 41653 6152 48180 42333 41754 35806 36233 14489 47594 5052 17544 4888 20218 8915 49660 5833 27173 8580 36604 22535 41532 4145 25576 30705 8116 11571 11044 18277 46343 6193 46215 49697 45360 12737 38565 42428 30395 45377 43359 12526 36496 34323 41281 38871 3847 42789 26360 28309 41492 32515 14001 30914 40581 29680 6517 5557 16788 477 26649 28531 48239 10119 10649 26718 8756 2177 14601 40415 35049 5543 4757 5730 26473 7907 11428 38118 45398 45091 11382 4856 39994 8039 44801 37794 46719 7640 33646 6186 31358 40671 12998 19150 46697 25659 24629 46732 7025 5371 6639 49593 12647 41349 25967 40820 4788 31482 43554 37310 10766 26690 17859 514 48558 18447 33188 44750 621 6164 5921 45546 39787 27526 13222 38259 42777 29913 40939 38170 37093 9427 49213 19116 31147 6421 42530 30077 11798 
199
40395 77235 95314 78058 98508 84643 86440 69384 6458 34327 39174 99762 41012 26405 30326 75919 52652 50142 82106 54821 10546 71331 71296 51827 79378 10204 44310 26234 87701 15059 79109 73319 35566 96343 44567 92765 71106 47962 27753 48643 9717 76091 3120 70386 53306 59741 74737 46985 83186 40769 15868 13483 82465 80852 52151 10516 50645 49559 48861 21848 15616 6375 26666 76582 38887 59157 29349 78026 99623 23244 11607 99857 34062 4011 52798 52609 69000 20585 87405 74515 97086 27646 54680 60878 23526 59880 68367 20239 49651 80504 83005 67362 24381 70098 39047 35510 37654 42094 27248 12122 42313 33037 88233 52514 36422 38110 9417 54838 14306 75108 29412 92096 55362 26471 23831 85162 96807 76986 75007 49767 13815 29573 71072 46593 18980 16723 77095 55740 35524 67350 76809 19115 73813 55806 96544 14652 93482 3001 74417 80884 69637 2492 871 13542 9406 39910 28860 20274 52352 23887 20864 23755 96912 96794 26073 69868 51894 68096 23085 29878 60996 81934 55618 10052 88511 56999 16175 89733 1676 46414 36222 71922 67945 68910 61185 41357 21927 85344 65666 83778 25531 31332 92072 7595 67445 41469 10168 91617 31484 96885 74010 99709 83279 8042 540 79606 90446 44472 86042 
43348 32206 7409 46620 6105 22304 711 36947 2031 1824 9984 18146 44986 38031 21113 28155 26749 17586 33007 13322 25804 26048 25706 33900 38651 42541 18219 1017 8582 9721 18396 32536 32494 17330 16206 28026 10421 21435 45359 7275 13283 29824 12876 24635 3140 27366 38514 13036 6169 33491 11607 13533 25968 36503 27841 4104 41015 9699 28379 20584 14193 27116 4600 35124 4406 49881 34872 39914 36749 37890 21274 14198 7271 17881 41232 3931 20703 13371 23947 44303 38310 21350 7815 3063 39830 12941 25161 1500 39539 12821 13793 29410 26788 23986 39692 7419 37631 33091 28524 44782 27453 40396 16977 26130 43249 36099 12923 36880 44164 39576 5991 8939 5528 5445 34331 35810 30804 10673 33465 15827 28640 35126 5451 29334 20418 14908 35991 10573 20038 37403 45024 8275 24875 33540 44151 21092 11845 28025 28570 9061 12435 44394 10083 1560 4325 41382 14875 37415 13975 7029 46480 14805 40340 46025 34648 45967 22274 35876 8632 10524 25360 30166 2607 49130 47916 18818 5148 33312 39545 33787 18464 35784 26824 25841 49268 9150 28059 26855 16962 41754 8940 18318 9656 25757 20754 28735 2720 34263 46779 48348 18240 49489 11816 44241 13250 11219 24798 47700 47819 
