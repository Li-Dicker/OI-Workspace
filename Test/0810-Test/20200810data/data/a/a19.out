Case #1: 14012842
Case #2: 13928378
Case #3: 13685918
Case #4: 14259716
Case #5: 13757450
Case #6: 13688208
Case #7: 15081399
Case #8: 14306797
Case #9: 14023634
Case #10: 14399125
